core/src/main/java/org/sonarsource/sonarlint/core/container/storage/ProtobufUtil.java

```diff
@@ -24,6 +24,7 @@
 import com.google.protobuf.Parser;
 import java.util.ArrayList;
 import java.util.List;
+import org.sonar.scanner.protocol.input.ScannerInput;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.scanner.protocol.input.ScannerInput'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)